client/my-sites/upgrades/domain-management/list/index.jsx

```diff
@@ -237,7 +241,10 @@ export const List = React.createClass( {
 	},
 
 	addDomainButton() {
-		if ( ! config.isEnabled( 'upgrades/domain-search' ) ) {
+		if (
+			! config.isEnabled( 'upgrades/domain-search' ) ||
+			this.props.selectedSite.jetpack
```

Comment:
Do we disable those buttons, because we can now only update domain-speific information, but not how it relates to the Jetpack site (add new one, change primary)?